src/c_kzg_4844.c

```diff
@@ -904,7 +904,7 @@ static void g1_lincomb(g1_t *out, const g1_t *p, const fr_t *coeffs, const uint6
     }
 }
 
-void blob_to_kzg_commitment(KZGCommitment *out, const Polynomial blob, const KZGSettings *s) {
```

Comment:
Unfortunately, using `Polynomial` here causes node-gyp to produce a C++ build error ("arg2 is not writable").

Reply:
I think I can revert this change now?